README.md

```diff
@@ -71,7 +71,7 @@ sudo dpkg -i fluent-plugin-logdna*.deb
 
 ## Additional Options
 
-For advanced configuration options, please refer to the [buffered output parameters documentation.](http://docs.fluentd.org/articles/output-plugin-overview#buffered-output-parameters)
+For advanced configuration options, please refer to the [buffered output parameters documentation.] (https://docs.fluentd.org/v/0.12/output#buffered-output-parameters)
```

Comment:
I think we still need the space between the `[]` and `()` to make the markdown correct.

Reply:
Thanks and good call Matt.  I tried to edit the last commit and it made a new commit instead.